src/library/simplified/extension/doubleclick-src-cat-type-PHIS.js

```diff
@@ -0,0 +1,306 @@
+if(b.siteId && utag.isPIS_FH() ){
```

Comment:
@trungnguyenexp I updated the check to  isPIS(), since it for all the PHIS page.